full-stack-engineer/submissions/michaelyao/server/README.md

```diff
@@ -0,0 +1,73 @@
+<p align="center">
```

Comment:
Server README are important. Would love to see your own README doc (instead of the default one)

Reply:
There is a READEME in this folder. But you are right, should copy or create a new one here too.
![Uploading image.png…]()

Reply:
Yes, I saw later on.  Removing this comment.